buildlib/pr/main.yml

```diff
@@ -263,6 +263,12 @@ stages:
         demands: ucx_roce -equals yes
         test_perf: 0
         proto_enable: yes
+    - template: tests.yml
+      parameters:
+        name: gpu_proto_enable
+        demands: ucx_gpu -equals yes
+        test_perf: 0
+        proto_enable: yes
```

Comment:
this will double the load on GPU machines and can lead to a longer time to get CI results

Reply:
IMO we should keep this PR open to fix the new CI failures

Reply:
if we won't merge this PR then maybe report failures as error?
I think we could add new variants to tests failed in this PR to cover related code.

Reply:
> if we won't merge this PR then maybe report failures as error?

Yes, let's do that